test/fixtures/node.yml

```diff
@@ -6,7 +6,6 @@ one:
   status: 1
   type: "note"
   uid: 2
-  nid: 1
```

Comment:
This change breaks the tests, so they have to be refactored not to rely on a hard `nid`. Please be sure tests pass in the future! I'm seeing if I can adjust them slightly and get tests passing again.